superbench/runner/ansible.py

```diff
@@ -87,7 +89,9 @@ def update_mpi_config(self, ansible_config):
         Returns:
             dict: Updated Ansible config dict.
         """
-        ansible_config['host_pattern'] += '[0]'
+        if not self._head_host:
+            logger.log_and_raise('no available host to launch mpi')
+        ansible_config['host_pattern'] = self._head_host
```

Comment:
if you raise error it will exit and won't run following benchmarks? could you just set a default value like "all[0]" instead of None?

Reply:
ok, updated